### PR TITLE
Fixing wizard validation

### PR DESCRIPTION
**Pull request by leog**
## What does this PR do?

Fixing wizard validation for https://github.com/calcom/console/issues/52

Also fixes https://github.com/calcom/console/issues/56

> **Warning**
> Please merge this before merging Console PR https://github.com/calcom/console/pull/55

**Environment**: Staging(main branch) / Production

## Type of change

- [X] Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

File: packages/ui/v2/core/WizardForm.tsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { useRouter } from "next/router";
 
+import { useLocale } from "@calcom/console/modules/common/hooks/useLocale";
```
Comment (leog) [inline packages/ui/v2/core/WizardForm.tsx:3]: This is to be able to use this component either from Web or Console

File: packages/ui/v2/core/WizardForm.tsx [hunk 2]
```diff
@@ -42,7 +42,7 @@ function WizardForm<T extends DefaultStep>(props: {
           <p className="text-sm text-gray-500">{currentStep.description}</p>
         </div>
 
-        <div className="print:p-none px-4 py-5 sm:p-6">{currentStep.content}</div>
+        <div className="print:p-none max-w-3xl px-4 py-5 sm:p-6">{currentStep.content}</div>
```
Comment (leog) [inline packages/ui/v2/core/WizardForm.tsx:45]: The fix for https://github.com/calcom/console/issues/56

File: apps/web/next.config.js [hunk 3]
```diff
@@ -15,6 +15,7 @@ const withTM = require("next-transpile-modules")([
   "@calcom/prisma",
   "@calcom/trpc",
   "@calcom/ui",
+  "@calcom/console",
```
Comment (leog) [inline apps/web/next.config.js:18]: This is to be able to import useLocale from `console` which in turn uses the one from web

File: apps/web/next.config.js [hunk 4]
```diff
@@ -3,6 +3,7 @@ const CopyWebpackPlugin = require("copy-webpack-plugin");
 const { withSentryConfig } = require("@sentry/nextjs");
 const os = require("os");
 const withTM = require("next-transpile-modules")([
+  "@calcom/console",
```
Comment (zomars) [inline apps/web/next.config.js:6]: Apps should never import between apps. I'm afraid I can't allow this.

File: packages/ui/v2/core/WizardForm.tsx (hunk 1 above)
Comment (zomars) [reply packages/ui/v2/core/WizardForm.tsx:3]: Why?

**Review by zomars — CHANGES_REQUESTED** [review]
Sorry @leog but this is breaking a rule of not importing between apps in the monorepo. That's what packages are for. Let's either extract the common logic to a package or find another way to fix the issue.

**Comment by zomars** [comment]
Also adding to this. This will break for ALL self-hosted users since they don't have access to the console repo.

**Comment by leog** [comment]
You are right, not my finest hour. I will find a solution, thanks @zomars

**Comment by zomars** [comment]
Closing for now

**Comment by leog** [comment]
@zomars better approach here: https://github.com/calcom/cal.com/pull/6068

**Pull request closed without merging**